fvm/transactionEnv.go

```diff
@@ -997,3 +997,15 @@ func (e *TransactionEnv) Commit() ([]programs.ContractUpdateKey, error) {
 	}
 	return e.contracts.Commit()
 }
+
+func (e *TransactionEnv) BLSVerifyPOP(pk *runtime.PublicKey, sig []byte) (bool, error) {
+	return crypto.VerifyPOP(pk, sig)
+}
+
+func (e *TransactionEnv) AggregateBLSSignatures(sigs [][]byte) ([]byte, error) {
+	return crypto.AggregateSignatures(sigs)
+}
+
+func (e *TransactionEnv) AggregateBLSPublicKeys(keys []*runtime.PublicKey) (*runtime.PublicKey, error) {
+	return crypto.AggregatePublicKeys(keys)
+}
```

Comment:
I wonder if we can prefix all of these methods with BLS and then add the method names (e.g. `BLSAggregateSignatures`) so we naturally have some grouping of these methods,